chaosazure/__init__.py

```diff
@@ -156,15 +159,19 @@ def __load_exported_activities() -> List[DiscoveredActivities]:
     return activities
 
 
-def __get_cloud_env_by_name(cloud=None):
+def __get_cloud_env_by_name(cloud: str = None) \
+        -> msrestazure.azure_cloud.Cloud:
     try:
         if cloud is None:
             return msrestazure.azure_cloud.AZURE_PUBLIC_CLOUD
-        elif cloud.strip() == "AZURE_CHINA_CLOUD":
+        else:
```

Comment:
You don't actually need that else because the if statement contains a return :)

```
if cloud is None:
     return msrestazure.azure_cloud.AZURE_PUBLIC_CLOUD

cloud = cloud.strip()
...
```